api/services/video.js

```diff
@@ -0,0 +1,25 @@
+const axios = require('axios').default;
+
+
+async function getVideoFileFromVimeo(video) {
+  if (video !== '') {
```

Comment:
Could simplify this if you want -- `if (video) {` would do basically the same job, checking it's not an empty/undefined string